lib/candies.test.js

```diff
@@ -16,5 +16,13 @@ it('Returns 404 not found on bad method', async () => {
   expect(text).toBe('Not Found');
 });
 
+it('Returns a 200 on #GET cats', async () => {
+  const { text, status } = await request(app)
+  .patch('/api/v1/cats')
```

Comment:
- `.patch` should be replaced with `.get` for this test
- expect would then change to look for a 200 status